pkgs/top-level/all-packages.nix

```diff
@@ -3399,6 +3399,8 @@ with pkgs;
 
   aws-env = callPackage ../tools/admin/aws-env { };
 
+  aws-gate = callPackage ../tools/admin/aws-gate { };
```

Comment:
Btw. we are going to replace `all-packages.nix` by a new `by-name` structure as detailed in <https://github.com/NixOS/rfcs/blob/c65c8321782b40844167beb48818471f70900d9d/rfcs/0140-simple-package-paths.md>, so you don't need to use `callPackage` here and simply move the `aws-gate` over to `pkgs/by-name/aw/aws-gate/package.nix`

Reply:
Oh, that's great to hear, will do it asap. Thank you for taking the time to review the PR 😸